app/javascript/components/program/MenuBar.js

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from "react";
+import { Link } from 'react-router-dom'
+import { Menu, Icon } from 'antd';
+
+const { SubMenu } = Menu;
+
+class MenuBar extends PureComponent {
+
+    constructor() {
+      super()
+      this.state = {
+
+      }
+    }
+
+    render() {
+      const { sections } = this.props;
+      const { party_acronym } = this.props;
+      const { section_id } = this.props;
+      const { selectedKey } = this.props;
+      const { openKey } = this.props;
+
+      return (
+        <Menu
+          mode="inline"
+          defaultSelectedKeys={selectedKey}
+          defaultOpenKeys={openKey}
```

Comment:
I can't make this work. When we are seeing a subsection I want to have the section menu open (check image below), but the entry is close but have blue color..

![Screenshot 2019-09-14 at 23 42 53](https://user-images.githubusercontent.com/38894857/64914436-72caf980-d749-11e9-8bb5-b80a3b2fee56.png)